pages/account/index.vue

```diff
@@ -30,18 +30,28 @@
               data-qa="public collections"
               :title="$t('account.publicCollections')"
             >
-              <!-- TODO: Update this section to preview the results retrieved with Sets API -->
-              <div class="text-center p-3">
-                Placeholder for {{ $t('account.publicCollections') }} tab.
+              <div v-if="!$fetchState.pending">
```

Comment:
The content in these tabs should have 40px distance from the tabs above.
There's some space in here now, but it's too little.